deptagency/algomart
# [#194] Authentication tests using Cypress

### Changes
- Cypress configuration updates
- [RTL support](https://www.npmjs.com/package/@testing-library/cypress) in Cypress tests
- Cypress commands that execute scripts for creating, removing, and updating a user in Firebase and the database
- Tests for auth flows:
  - Logging in as an existing user
  - Signing out when logged in
  - Creating a new account
- New API endpoint for removing a user

Closes #194 

### Testing
- Run `npm run e2e` to start up the application
- The Cypress window will appear eventually
- Click to Run all tests
- Observe test suite run

### Next up
- [ ] #193 
- [ ] #198
